README.md

```diff
@@ -1 +1,137 @@
-# promtracker
\ No newline at end of file
+# Motoko value tracker for prometheus
+
+## Overview
+
+A value tracker, designed specifically to use as source for Prometheus.
+It contains functionality to register/deregister various kinds of custom values on the fly, 
+built-in system stats of your canister, and statistics output as `Text` in prometheus exposition format
+
+## Links
+
+The package is published on [MOPS](https://mops.one/promtracker) and [GitHub](https://github.com/research-ag/promtracker).
+
+The API documentation can be found [here](https://mops.one/promtracker/docs/lib) on Mops.
+
+For updates, help, questions, feedback and other requests related to this package join us on:
+
+* [OpenChat group](https://oc.app/2zyqk-iqaaa-aaaar-anmra-cai)
+* [Twitter](https://twitter.com/mr_research_ag)
+* [Dfinity forum](https://forum.dfinity.org/)
+
+## Examples
+
+Look at our [example canister](examples/example_canister.mo)
+
+Create tracker instance like this:
+```motoko
+let tracker = PT.PromTracker(65);
+```
+65 seconds is the recommended interval if prometheus pulls stats with interval 60 seconds. This value used to clear high 
+and low watermarks in [gauge values](#gauge-value), so each highest and lowest value during your canister lifecycle will
+be reflected in prometheus graphs
+
+Add some values:
+```motoko
+let successfulHeartbeats = tracker.addCounter("successful_heartbeats", true);
+let failedHeartbeats = tracker.addCounter("failed_heartbeats", true);
+let heartbeats = tracker.addPullValue("heartbeats", func() = successfulHeartbeats.value() + failedHeartbeats.value());
+let heartbeatDuration = tracker.addGauge("heartbeat_duration", true);
```

Comment:
For a gauge the stable flag would usually be false, or?

Reply:
As user prefers, but probably yes

Reply:
removed the flag completely